testing/common.sh

```diff
@@ -128,9 +125,6 @@ function install_operator_version() {
   # Initialize the MinIO Kubernetes Operator
   kubectl minio init
 
-  echo "Scaling down MinIO Operator Deployment"
-  try kubectl -n minio-operator scale deployment minio-operator --replicas=1
```

Comment:
Q: Why is this removed?

Reply:
Because it is not needed, we have two Operator replicas and that is perfectly fine and the reason they are reducing this to one is making test more prone to failure as we are not even waiting for the proper termination of this pod. So better to not have this line if our test is not reliable because of it.